views/install.sh.erb

```diff
@@ -290,7 +292,7 @@ capture_tmp_stderr() {
 # do_wget URL FILENAME
 do_wget() {
   echo "trying wget..."
-  wget -O "$2" "$1" 2>$tmp_dir/stderr
+  wget -t 1 -T $download_timeout -O "$2" "$1" 2>$tmp_dir/stderr
```

Comment:
I think if we make the default to not set a timeout that I'll buy your argument to set them all.

As it stands, though, if we're solving the "offline" problem then the dns-timeout will be what I care about, and the read-timeout will only annoy me when I have a connection but something hiccups in my network for 10 seconds (e.g. spanning tree does its thing because I lost a router and my network was designed in 2003 and I haven't figured out how to configure RSTP).